src/main/java/com/hazelcast/remotecontroller/HazelcastCloudManager.java

```diff
@@ -0,0 +1,338 @@
+        try {
+            String clusterName = "test-cluster-" + System.currentTimeMillis();
+            String query = String.format("{\"query\":\"mutation {createStarterCluster(input: {name: \\\"%s\\\" cloudProvider: \\\"%s\\\" region: \\\"%s\\\" clusterType: SMALL totalMemory: 2 hazelcastVersion: \\\"%s\\\" isTlsEnabled: %b } ) { id name hazelcastVersion isTlsEnabled state discoveryTokens {source,token} } }\"}",
+                    clusterName,
+                    "aws",
+                    "us-west-2",
+                    hazelcastVersion,
+                    isTlsEnabled);
+
+            Log.info(String.format("Request query: %s", query));
+            String responseBody = createRequest(query).body().string();
+            LOG.info(maskValueOfToken(responseBody));
+            JsonNode rootNode = mapper.readTree(responseBody).get("data").get("createStarterCluster");
+            if(!waitForStateOfCluster(rootNode.get("id").asText(), "RUNNING", TimeUnit.MINUTES.toMillis(5)))
+                throw new Exception("Wait for cluster state is not finished as expected");
+            return getHazelcastCloudCluster(rootNode.get("id").asText());
+        } catch (Exception e) {
+            e.printStackTrace();
+            return null;
+        }
+    }
+
+    public boolean scaleUpDownHazelcastCloudStandardCluster(String clusterId, int scaleNumber) {
+        int currentMemory;
+        String requestUrl = String.format("%s/cluster/%s/updateMemory", baseUrl, clusterId);
+        try {
+            String query = String.format("{ \"query\": \"query { cluster(clusterId: \\\"%s\\\") { specs {totalMemory} } }\" }", clusterId);
+            String responseBody = createRequest(query).body().string();
+            currentMemory = mapper.readTree(responseBody).get("data").get("cluster").get("specs").get("totalMemory").asInt();
+
+            if (currentMemory + scaleNumber <= 0) {
+                LOG.warn("Scaling number is not proper, there is no enough memory and nodes to scale down");
+                return false;
+            }
+
+            String requestBody = String.format("{\"id\": %s, \"memory\": %d, \"autoScalingEnabled\": false}", clusterId, currentMemory + scaleNumber);
+            RequestBody body = RequestBody.create(JSON, requestBody);
+            Request request = new Request.Builder()
+                    .url(requestUrl)
+                    .post(body)
+                    .header("Content-Type", "application/json")
+                    .header("Authorization", String.format("Bearer %s", bearerToken))
+                    .build();
+            call = client.newCall(request);
+            Response response = call.execute();
+            responseBody = response.body().string();
+            LOG.info(mapper.readTree(responseBody).get("state"));
+            waitForStateOfCluster(clusterId,"RUNNING", TimeUnit.MINUTES.toMillis(1));
```

Comment:
Scaling is generally takes around 25-30 seconds for standard cluster. The `scaleNumber` parameter is to change of the node count. For example if you have a cluster with 2 nodes and you send a request to scale the cluster with `scaleNumber=2` it will scale up the cluster to 4 nodes. If you send the scale request with `scaleNumber=-1` it will close 1 node and your cluster will be 1 node.